index.php

```diff
@@ -19,7 +19,9 @@
 	$start_index = jobs_start_index_for_page( $page_num );
 }
 
-$jobs_list = get_jobs( $start_index, $search ); ?>
+$status = "active";
+
+$jobs_list = get_jobs( $start_index, $status, $search ); ?>
```

Comment:
Why don't you just pass `'active'` to the function?
```suggestion
$jobs_list = get_jobs( $start_index, 'active', $search ); ?>
```